src/commands/others/pin.ts

```diff
@@ -0,0 +1,80 @@
+import {
+    ApplicationCommandType,
+    ContextMenuCommandBuilder,
+    InteractionContextType,
+    type MessageContextMenuCommandInteraction,
+} from "discord.js";
+
+import type { DatadropClient } from "../../datadrop.js";
+import type { Command } from "../../models/Command.js";
+
+export default {
+    data: new ContextMenuCommandBuilder()
+        .setName("(Dés)Épingler le message")
+        .setType(ApplicationCommandType.Message)
+        // .setDescription("(Dés)épingle le message sélectionné dans le canal.")
```

Comment:
Wouldn't it be better to delete this line of code instead of commenting it?

Reply:
You are right! Thanks for pointing it out :)
```suggestion
```